src/RestApi/Controllers/ProductCategories.php

```diff
@@ -117,6 +117,27 @@ public function prepare_item_for_response( $item, $request ) {
 			'permalink'   => get_term_link( $item->term_id, 'product_cat' ),
 		);
 
+		if ( $request->get_param( 'show_review_count' ) ) {
```

Comment:
We'll need to add `show_review_count` to the collection params as part of the schema for this endpoint.